### PR TITLE
Johnny

### PR DESCRIPTION
**Pull request by GrfxGuy79**

### CODE REVIEW

File: tour.html [hunk 1]
```diff
@@ -4,86 +4,101 @@
...
+				</ul>
+			</nav>
+		</header>
 
 			<!-- MAIN -->
 			<main class="dates">
-                <h1>Tour Dates</h1>
-                <h2>Locations</h2>
-                
-				<section class="locationsDesktop">
```
Comment (jaleguizamon) [inline tour.html:58]: why was the locationsDesktop class name removed? A few other class names were altered. What was the reasoning behind that?

File: tour.html [hunk 2]
```diff
@@ -4,86 +4,101 @@
...
+						<a href="individualEvent.html?city=atl&date=6/22/22"
+							>June 22, 2022<span class="tickets">Tickets &#8250;</span></a
+						>
+						<a href="individualEvent.html?city=atl&date=2022-06-23"
+							>June 23, 2022<span class="tickets">Tickets &#8250;</span></a
+						>
+					</div>
+				</section>
 
-                <section class="locationsMobile">
```
Comment (jaleguizamon) [inline tour.html:82]: why was this section entirely deleted? This has functionality on the page

**Review by jaleguizamon — CHANGES_REQUESTED** [review]
I am not approving this as my code, seems to have been altered. Class names that provided functionality and entire sections of my HTML were removed. Please clarify on these decisions

**Comment by GrfxGuy79** [comment]
Fixed errors

**Pull request closed without merging**